app/src/main/res/values/strings.xml

```diff
@@ -204,6 +210,10 @@
     <string name="first_run_page4_text">Take screenshot of an entire page scroll. Read it offline without using data or tap it to revisit the web page.</string>
     <string name="first_run_page5_title">Next-level privacy</string>
     <string name="first_run_page5_text">Browse without a trace, and block a wide range of common web trackers by default. Ensure the security of your confidential data.</string>
+    <string name="first_run_page6_title">Discover new things</string>
+    <!-- The %3$s here is a placeholder for "Life Feed"-->
+    <string name="first_run_page6_text">Just one flip, keep yourself updated with the “%3$s” - a variety of contents from the most popular topics to the hottest deals.</string>
```

Comment:
Just one flip, keep yourself updated with the “%1$s” - a variety of contents from the most popular topics to the hottest deals. %2$s

Reply:
We need that %2$s as a placeholder for learnmore. Just like:
    <string name="first_run_upgrade_page_text1">1. Multitask by opening %s at the same time!</string>
    <string name="first_run_upgrade_page_text1_highlight">multiple tabs</string>